var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -67,13 +68,35 @@ def install(self, spec, prefix):
         options = [
             '--prefix=%s' % prefix,
             '--enable-optimize',
-            '--enable-tools',
+            '--disable-tools',
+            '--disable-mbconvert',
+            '--disable-hexmodops',
+            '--disable-vtkMOABReader',
+            '--disable-mbsize',
+            '--disable-mbskin',
+            '--disable-mbtagprop',
+            '--disable-mbmem',
+            '--disable-spheredecomp',
+            '--disable-mbsurfplot',
+            '--disable-mbpart',
+            '--disable-dagmc',
+            '--disable-gsets',
+            '--disable-mbmerge',
+            '--disable-mbdepth',
+            '--disable-mbcoupler',
+            '--disable-mcnpmit',
+            '--disable-refiner',
+            '--disable-h5mtools',
+            '--disable-mbcslam',
+            '--disable-mbquality',
+            '--disable-ahf',
+            '--disable-mbumr',
+            '--disable-imesh',
```

Comment:
> . With most build systems

You mean, with Autotools.  CMake has tried to educate the world about resisting the temptation to auto-configure based on what's available.